src/mono/netcore/CoreFX.issues.rsp

```diff
@@ -234,7 +234,6 @@
 -nomethod System.Reflection.Emit.Tests.MethodBuilderToString.ToString_NameAttributeAndSignatureSetSet
 -nomethod System.Reflection.Emit.Tests.MethodBuilderToString.ToString_NonGenericMethod
 -nomethod System.Reflection.Emit.Tests.MethodBuilderToString.ToString_GenericMethod
--nomethod System.Reflection.Emit.Tests.ModuleBuilderDefineEnum.DefineEnum
```

Comment:
note that this file was removed so you need to rebase and remove the `ActiveIssue` here instead:

https://github.com/dotnet/runtime/blob/6a33d96168c2f7ae3f9e803fa646d9c3bb821488/src/libraries/System.Reflection.Emit/tests/ModuleBuilder/ModuleBuilderDefineEnum.cs#L33-L35